common/co_dispatcher.py

```diff
@@ -71,6 +73,35 @@ def __init__(self,
         # Exception to be injected into that task at current `yield`.
         self._to_raise = None
 
+    @property
+    def traceback_lines(self):
+        task = self
+        e = task.exception
+
+        if e is None:
+            return []
+
+        lines = []
+
+        while isinstance(e, (CancelledCallee, FailedCallee)):
+            if task is not None:
```

Comment:
Can there be a situation when the `e.callee` is `None`?

Reply:
Normally not. If so, it's an error somewhere else.